Tupfile

```diff
@@ -0,0 +1,3 @@
+include_rules
+
+: cv.tex	|> !xelatex	|> %B.pdf %B.out %B.aux %B.log
```

Comment:
The whole point of the makefile was to build in a different folder, so that the log files do not pollute the source folder